src/utils/convertExperimentToApiV1Model.js

```diff
@@ -0,0 +1,20 @@
+// There are some differences between the property naming of elements
+// stored in the ui and in the api,
+// this is an attempt to deal with this in one single place (the ui)
+// We should try to converge to one single model to follow
+const convertExperimentToApiV1Model = (experiment) => {
```

Comment:
Will there be any more models that need conversion for v1 to v2? If there are, maybe better make a folder to store these functions or export this function as a named export so people can extend easily

Reply:
No, this isn't converting v1 to v2

Reply:
It was added a couple of months ago already, it's about the differences between the experiment in redux vs the one being stored in dynamo

Reply:
Also it will no longer be necessary when we get rid of v1